models/baseModel.js

```diff
@@ -0,0 +1,29 @@
+const Sequelize = require('sequelize');
+
+// Sequelize automatically adds 'createdat' and 'updatedat' fields:
+// http://docs.sequelizejs.com/manual/tutorial/models-definition.html#timestamps
+// but keep in mind we'll need to handle those fields on our own if we
+// add a migration
+module.exports = class BaseModel extends Sequelize.Model {
+    /**
+     * Initialize this model
+     * @param childCols the field information of a subclass
+     * of this model that should be merged with the fields
+     * defined here
+     * @param sequelize the sequelize object
+     */
+    static init(childCols, sequelize) {
+        const baseCols = {
+            buildNum: {
+                type: Sequelize.INTEGER,
+                primaryKey: true
+            },
+            buildUrl: {
+                type: Sequelize.STRING
+            }
+        };
+        const allCols = Object.assign(baseCols, childCols);
+
+        return super.init(allCols, sequelize);
```

Comment:
Is this the normal pattern and naming convention for sequelize? BaseClass is not intended to be initialized on its own?

Reply:
`BaseModel` isn't actually a table, it just contains common fields that other tables contain, so it doesn't get initialized on its own.

as to whether it's a common pattern/naming convention, i don't know.  the inheritance is pretty new to sequelize and there aren't many examples.  i wanted to use it though since there was data shared amongst the tables that i didn't want to duplicate, and i think as we add more charts we'll leverage this kinda of re-use more and more.